lib/fluent/plugin/in_tail.rb

```diff
@@ -593,13 +592,11 @@ def on_notify
           begin
             if @inode != inode || fsize < @fsize
               # rotated or truncated
+              io = FileWrapper.open(@path)
```

Comment:
When Errno::ENOENT occurs on `FileWrapper.stat(@path)`, it seems Errno::ENOENT occurs again at here. Do not you need to handle it?